workspace/notebook/py_unit_tests_service_user.json

```diff
@@ -598,7 +694,7 @@
 					"SELECT * \n",
 					"FROM\n",
 					"odw_curated_db.nsip_representation AS reps \n",
-					"INNER JOIN odw_curated_db.service_user AS suser\n",
+					"INNER JOIN odw_curated_db.appeal.service_user AS suser\n",
```

Comment:
@Syed-KC  is this supposed to be `odw_curated_db.appeal_service_user` instead?

Reply:
good catch! yeah it is supposed to be that, is it easy for you to change or shall I do a new commit?

Reply:
I'll update it no problem